### PR TITLE
Wallets, Widgets and tests

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: packages/wallets/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@lidofinance/wallets-testing-wallets",
+  "version": "0.0.0-semantic-release",
```
Comment (dgusakov) [inline packages/wallets/package.json:3]: Should it be 0.0.0?

File: packages/wallets/src/coin98/coin98.page.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    await inputs.nth(1).fill(this.config.PASSWORD);
+    await this.page.locator('button:has-text("Setup Password")').click();
+    await this.page.locator('button:has-text("Ok")').click();
+    await this.page.locator('[placeholder="Search chain"]').fill('eth');
+    await this.page.locator('.box-logo').click();
+    await this.page.locator('[placeholder="Wallet name"]').fill('test');
+    await this.page
+      .locator('.content-editable--password')
+      .fill(this.config.SECRET_PHRASE);
+    await this.page.locator('button:has-text("Restore")').click();
```
Comment (dgusakov) [inline packages/wallets/src/coin98/coin98.page.ts:44]:
```suggestion
  async firstTimeSetup() {
    if (!this.page) throw "Page isn't ready";
    await this.page.click('text=Restore Wallet');
    await this.page.click('text=Ok');
    const inputs = this.page.locator('input[type=password]');
    await inputs.nth(0).fill(this.config.PASSWORD);
    await inputs.nth(1).fill(this.config.PASSWORD);
    await this.page.click('button:has-text("Setup Password")');
    await this.page.click('button:has-text("Ok")');
    await this.page.fill('[placeholder="Search chain"]', 'eth');
    await this.page.click('.box-logo');
    await this.page.fill('[placeholder="Wallet name"]', 'test');
    await this.page
      .fill('.content-editable--password', this.config.SECRET_PHRASE);
    await this.page.click('button:has-text("Restore")');
```

File: packages/wallets/src/metamask/metamask.page.ts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+  async addNetwork(networkName: string, networkUrl: string, chainId: string) {
+    if (!this.page) throw "Page isn't ready";
+    await this.page.click('.account-menu__icon');
+    await this.page.click('text=Settings');
+    await this.page.click("text='Networks'");
+    await this.page.click('text=Add a network');
+    await this.page
+      .locator(".form-field :has-text('Network Name') >> input")
+      .fill(networkName);
```
Comment (dgusakov) [inline packages/wallets/src/metamask/metamask.page.ts:77]: Hereandafter replace locator(a).fill(b) with .fill(a,b)

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: packages/wallets/package.json (hunk 1 above)
Comment (skhomuti) [reply packages/wallets/package.json:3]: yep, it's just placeholder and actual version will be taken from git tags

File: packages/wallets/src/metamask/metamask.page.ts (hunk 3 above)
Comment (skhomuti) [reply packages/wallets/src/metamask/metamask.page.ts]: fixed

File: packages/wallets/src/coin98/coin98.page.ts (hunk 2 above)
Comment (skhomuti) [reply packages/wallets/src/coin98/coin98.page.ts]: fixed

**Review by dgusakov — APPROVED** [review]
Need to find out why tests were failed

**Pull request merged**